docker-compose.yaml

```diff
@@ -24,6 +24,20 @@ services:
       retries: 3
       start_period: 10s
 
+  pgadmin:
+    container_name: pgadmin
```

Comment:
Safe to pin this to "latest" ? Or should we be more strict?

Reply:
Hard to say. I don't have long-term experience with this image as previously I had pgadmin just installed on my system. As this is a development tool and read-only at that, I don't think it's an issue and I think we can go with latest.